tests/test_vrf.rb

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) 2015 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative 'ciscotest'
+require_relative '../lib/cisco_node_utils/vrf'
+
+include Cisco
+
+# TestVrf - Minitest for Vrf node utility class
+class TestVrf < CiscoTestCase
+  VRF_NAME_SIZE = 33
+
+  def setup
+    super
+    vrfs = Vrf.vrfs
+    vrfs.each_value do |vrf|
+      next unless vrf.name =~ /^test_vrf/
+      @device.cmd('conf t')
+      @device.cmd("no vrf context #{vrf.name}")
+      node.cache_flush
+    end
+  end
+
+  def test_vrf_collection_not_empty
+    vrfs = Vrf.vrfs
+    refute_empty(vrfs, 'VRF collection is empty')
+    assert(vrfs.key?('management'), 'VRF management does not exist')
```

Comment:
Consider changing to:

`assert_includes(vrfs, 'management', 'VRF management does not exist')`